test/tests/gcc-c-hello-world/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+image="$1"
+dirTest="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+dirContainer='/usr/src/c'
+
+docker run --rm -v "$dirTest":"$dirContainer":rw -w "$dirContainer" "$image" gcc -o hello-world  hello-world.c
+"$dirTest"/hello-world
+rm -rf "$dirTest"/hello-world
```

Comment:
I would rather have the compiled file stay inside the container, that way we don't need the `rm` or the `rw` mount.
```
docker run --rm -v "$dirTest":"$dirContainer":ro -w "$dirContainer" "$image" sh -c 'gcc -o /hello-world hello-world.c && /hello-world'
```